prime-angular-frontend/src/app/modules/site-registration/pages/business-licence-page/business-licence-form-state.class.ts

```diff
@@ -89,14 +90,21 @@ export class BusinessLicenceFormState extends AbstractFormState<BusinessLicenceF
     });
   }
 
+  public patchDocument(document: BusinessLicenceDocument): void {
+    if (!this.businessLicence) {
+      return;
+    }
+    this.businessLicence.businessLicenceDocument = document;
+  }
```

Comment:
can this be changed to:


```
  public patchDocument(document: BusinessLicenceDocument): void {
    if (this.businessLicence) {
      this.businessLicence.businessLicenceDocument = document;
    }
  }
```

Reply:
This is an alternate way to write it, and currently our other patch methods fail out so I continued that trend with this patch method.